fpsim/locations/nuhdss.py

```diff
@@ -0,0 +1,1252 @@
+'''
+Set the parameters for FPsim, specifically for NUHDSS.
+'''
+
+import numpy as np
+import pandas as pd
+import sciris as sc
+from scipy import interpolate as si
+from .. import defaults as fpd
+from .. import utils as fpu
+
+# %% Housekeeping
+
+thisdir = sc.thispath(__file__)  # For loading CSV files
+
+
+def scalar_pars():
+    scalar_pars = {
+        'location':             'nuhdss',
+        'postpartum_dur':       23, #We used the median #We do  have. Askwhat to calculate!
```

Comment:
Good question! This parameter just tells the model how long women are considered to be "postpartum" - it will be the same for all women, and will affect their sexual activity and FP. You can use any value (in months) that you like.